.travis/.travis_set_deploy_build_opts.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -x
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+if [ "$TRAVIS_EVENT_TYPE" = "cron" ]; then
+  export DEV_VERSION="-dev-${TRAVIS_BUILD_NUMBER}"
```

Comment:
I asked @varidhigarg to verify if versions like 0.4.0-dev-1, 0.4.0-dev-2 will work with auto-ELR. We believe it should work so please don't block this PR on this verification - it should not be a blocker for merging this in. Thanks!!

Reply:
As long as the mavenmetadata.xml is updated with the latest version we are good to go.